### ISSUE TITLE
test_copy_from_propagate_error fails with Python 3.5

### ISSUE THREAD

**Issue by desiderius** [psycopg/psycopg2#412]
Hello, I am trying to update the package of psycopg2 (from the source on PyPI) on `NixOS/Linux` but the following unit test fails with Python 3.5.1:

```
======================================================================
FAIL: test_copy_from_propagate_error (psycopg2.tests.test_copy.CopyTests)
----------------------------------------------------------------------
ZeroDivisionError: division by zero

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/nix/store/mdwpl1s8iwpj06r5zvva3bfhr7lswwfp-python3.5-psycopg2-2.6.1/lib/python3.5/site-packages/psycopg2/tests/test_copy.py", line 356, in test_copy_from_propagate_error
    curs.copy_from(BrokenRead(), "tcopy")
SystemError: <class 'psycopg2.extensions.QueryCanceledError'> returned a result with an error set

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/nix/store/mdwpl1s8iwpj06r5zvva3bfhr7lswwfp-python3.5-psycopg2-2.6.1/lib/python3.5/site-packages/psycopg2/tests/testutils.py", line 291, in skip_if_green__
    return f(self)
  File "/nix/store/mdwpl1s8iwpj06r5zvva3bfhr7lswwfp-python3.5-psycopg2-2.6.1/lib/python3.5/site-packages/psycopg2/tests/test_copy.py", line 358, in test_copy_from_propagate_error
    self.assertTrue('ZeroDivisionError' in str(e))
AssertionError: False is not true

----------------------------------------------------------------------
Ran 501 tests in 18.250s

FAILED (failures=1, skipped=53)
```

I would like to confirm whether this is an expected error (in the current state of the sources on PyPi) or a mistake in my packaging.

**Comment by dvarrazzo** [comment]
We haven't tested psycopg with python 3.5 yet.

**Comment by desiderius** [comment]
Thank you for the feedback, can I help with the test with Python 3.5 in a way ?

**Comment by dvarrazzo** [comment]
Fixed, to be released in 2.6.2
